mono/mini/test_op_il_seq_point.sh

```diff
@@ -6,7 +6,12 @@ USE_AOT=$2
 TMP_FILE_PREFIX=$(basename $0).tmp
 BASEDIR=$(dirname $0)
 
-MONO_PATH=$BASEDIR/../../mcs/class/lib/net_4_x:$BASEDIR
+case "$(uname -s)" in
+	CYGWIN*) SEP=';';;
+	*) SEP=':';;
```

Comment:
We're calling this PLATFORM_PATH_SEPARATOR in other parts of the build system, would be good to be consistent.